### PR TITLE
[13.0][FIX][stock_valuation_layer_usage]: look for true incoming moves

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
In case that the out move is connected with an internal transfer,
no valuation layer will be found, and then the system will create
a layer with negative quantity. We should try to look for the
true incoming move, if there is any.

cc @ForgeFlow

Backport from v14: https://github.com/OCA/stock-logistics-workflow/pull/1004

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
CI fails because of an import in other module :confused: 

```
workflow/stock_picking_group_by_partner_by_carrier_by_date/tests/__init__.py", line 1, in <module>
    from . import test_grouping_by_date
  File "/__w/stock-logistics-workflow/stock-logistics-workflow/stock_picking_group_by_partner_by_carrier_by_date/tests/test_grouping_by_date.py", line 3, in <module>
    from freezegun import freeze_time
ModuleNotFoundError: No module named 'freezegun'
```

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

File: stock_valuation_layer_usage/models/stock_move.py [hunk 1]
```diff
@@ -46,3 +46,12 @@ def _create_dropshipped_svl(self, forced_quantity=None):
                 }
             )
         return layers
+
+    def _get_in_moves(self):
```
Comment (rousseldenis) [inline stock_valuation_layer_usage/models/stock_move.py:50]: I'd prefer returning ids as better in terms of performances to do a set of ids and not using `|` operator

**Comment by rousseldenis** [comment]
@AaronHForgeFlow

**Review by JordiMForgeFlow — APPROVED** [review]

**Pull request closed without merging**